### ISSUE TITLE
Crash when entering MystCraft age

### ISSUE THREAD

**Issue by 3DLudi** [XCompWiz/Mystcraft-Issues#183]
I am trying to get the new 1.11.2 MystCraft working but I seem to be doing a poor job of it.

I configured it to profile during load.  It displays a fake world with a progress bar.  Takes a while but no issues here.  When it finishes I create a new world, create a blank age (for testing) but when I click the age preview (which shows a broken texture) to enter the age it crashes.  Thought it might be some specific generation causing it so I tried four or five times but it never actually puts me in the world.

I have a pastebin of one of the crash logs but I am terrible at reading them.
All I see is: **blah blah blah** MystCraft **blah blah blah** Chisel **blah blah blah** Savanna Biome

Could someone please take a peek at it and point me to any probable causes?
Thanks!

https://pastebin.com/y4Z5fyPh

**Comment by velhonoja** [comment]
I have similar crashes when entering ages. 

I tried test world with only mystcraft mod, and it works.

I also tried to find other mod that is causing trouble with mystcraft, but crashing seems quite random. For example removing forestry fixed some crashing, but not all (and sometimes it works with forestry).

This exception is usually on crash reports "java.lang.IllegalArgumentException: Cannot add types to unregistered biome com.xcompwiz.mystcraft.world.biome.BiomeWrapperMyst"
